### PR TITLE
62 refactor service types

### PR DESCRIPTION
**Pull request by jebbinSCL**
Merge and release before front/ 62 refactor service types

### CODE REVIEW

File: Speakers/Controllers.fs [hunk 1]
```diff
@@ -2,17 +2,17 @@
 
 open System
```
Comment (lclarksl) [inline Speakers/Controllers.fs]: Do we need open System here?

Comment (jebbinSCL) [inline Speakers/Controllers.fs]: Dealt with

**Comment by thullSL** [comment]
Apart from the naming of "talkOutline" :shipit:

**Comment by lclarksl** [comment]
:shipit:

**Pull request merged**